### PR TITLE
What files should be committed or ignored

### PR DESCRIPTION
**Pull request by citizenmatt**
Added an FAQ to make it clear what files should be committed or ignored from source control.

Complete newbie at paket, so not sure I've got it right :)

### CODE REVIEW

File: docs/content/faq.md [hunk 1]
```diff
@@ -65,6 +65,22 @@ There is no reasonable way to make this work – and even NuGet.exe can't do it
...
+
+* [`paket.dependencies`](dependencies-file.html) - specifies your application's dependencies, and how they should be fulfilled.
+* All [`paket.references`](references-files.html) files - each project will have a `paket.references` file that specifies which of the dependencies are installed in the project. Each of these files should be committed to source control.
+* [`paket.lock`](lock-file.html) - records the actual versions used during resolution. If it exists, Paket will ensure that the same versions are used when restoring packages. It is not strictly necessary to commit this file, but strongly recommended (see the following question for details why).
+
+The following files can be committed, but are not essential:
+
+* [`.paket/paket.bootstrapper.exe`](getting-started.html) - this is a small, rarely updated executable that will download the latest version of the main `paket.exe`. If you prefer to place `paket.exe` in the path, or use a [Paket plugin for your editor](editor-support.html), then you don't need to commit this file. However, not committing it also requires that other users of your solution have `paket.exe` available.
```
Comment (forki) [inline docs/content/faq.md]: others and build servers too

**Comment by forki** [comment]
Awesome!

File: docs/content/faq.md (hunk 1 above)
Comment (citizenmatt) [inline docs/content/faq.md]: I didn't see any docs about how build servers would use it - is it just an assumption for scripts?

Comment (forki) [inline docs/content/faq.md]: Many users have a build.sh or buld.cmd with:

    .paket/paket.bootstrapper.exe
    .paket/paket.exe restore
    *run whatever build tool*

paket itself has this as well

**Comment by forki** [comment]
Very cool. I will also add some words about Paket.targets which allows you to restore in VS  without external build script.

**Comment by forki** [comment]
Thanks so much

**Pull request merged**